src/graph/validator/MaintainValidator.cpp

```diff
@@ -59,6 +60,12 @@ static Status validateColumns(const std::vector<ColumnSpecification *> &columnSp
     if (!column.nullable_ref().has_value()) {
       column.nullable_ref() = true;
```

Comment:
I found this problem recently as well, do you know why? This is quite implicit...

Reply:
Is there any related issue? I don't know which problem you're talking about.

Reply:
> I found this problem recently as well, do you know why? This is quite implicit...

Is there any related issue? I don't know which problem you're talking about.